docs/writing-shaders.md

```diff
@@ -22,7 +22,15 @@ follow deck.gl's projection methods, enabling your layer to accept coordinates
 in both [longitude,latitude,altitude] or [metersX,metersY,metersZ] format.
 
 The projection package is included by default by the `assembleShaders` function,
-and offers three functions `project_position`, `project_scale` and `project_to_clipspace`.
+and offers the following functions:
+
+#### `project_position`
+
+#### `project_scale`
+
+#### `project_to_viewspace`
+
```

Comment:
Feels a bit strange not to be able to go to clipspace from viewspace but we can build this out later by adding the raw projectionMatrix.